packages/sdk/src/controllers/VariableController.ts

```diff
@@ -290,6 +290,118 @@ export class VariableController {
+     * @param minimum the minimum value or null to remove the minimum value
+     * @returns
+     */
+    setMinimum = async (id: string, minimum: number | null) => {
+        const res = await this.#editorAPI;
+        const update = { min: minimum };
+        return res
+            .updateNumberVariableProperties(id, JSON.stringify(update))
+            .then((result) => getEditorResponseData<null>(result));
+    };
+
+    /**
+     * @experimental Sets the maximum value for a number variable
+     * @param id the id of the variable to update
+     * @param maximum the maximum value or null to remove the maximum value
+     * @returns
+     */
+    setMaximum = async (id: string, maximum: number | null) => {
+        const res = await this.#editorAPI;
+        const update = { max: maximum };
+        return res
+            .updateNumberVariableProperties(id, JSON.stringify(update))
+            .then((result) => getEditorResponseData<null>(result));
+    };
+
+    /**
+     * @experimental Sets the visibility of the stepper for a number variable
+     * @param id the id of the variable to update
+     * @param showStep true to show the stepper, false to hide it
+     * @returns
+     */
+    setShowStepper = async (id: string, showStepper: boolean) => {
+        const res = await this.#editorAPI;
+        const update = { showStepper: showStepper };
+        return res
+            .updateNumberVariableProperties(id, JSON.stringify(update))
+            .then((result) => getEditorResponseData<null>(result));
+    };
+
+    /**
+     * @experimental Sets the step size for a number variable
+     * @param id the id of the variable to update
+     * @param stepSize the step size or null to remove the step size
+     * @returns
+     */
+    setStepSize = async (id: string, stepSize: number) => {
+        const res = await this.#editorAPI;
+        const update = { stepSize: stepSize };
+        return res
+            .updateNumberVariableProperties(id, JSON.stringify(update))
+            .then((result) => getEditorResponseData<null>(result));
+    };
+
+    /**
+     * @experimental Sets the thousands separator for a number variable
+     * @param id the id of the variable to update
+     * @param thousandsSeparator the thousands separator to use
+     * @returns
+     */
+    setThousandsSeparator = async (id: string, thousandsSeparator: string) => {
+        const res = await this.#editorAPI;
+        const update = { thousandsSeparator: thousandsSeparator };
+        return res
+            .updateNumberVariableProperties(id, JSON.stringify(update))
+            .then((result) => getEditorResponseData<null>(result));
+    };
+
+    /**
+     * @experimental Sets the decimal separator for a number variable
+     * @param id the id of the variable to update
+     * @param decimalSeparator the decimal separator to use
+     * @returns
+     */
+    setDecimalSeparator = async (id: string, decimalSeparator: string) => {
```

Comment:
@chili-publish/react thoughts about the following?

```suggestion
    setDecimalSeparator = async (id: string, decimalSeparator: '' | '.' | ',' | ' ') => {
```